src/tests/test_aladin.py

```diff
@@ -130,3 +132,71 @@ def test_aladin_angle_fov_set(angle: float) -> None:
     angle_fov = Angle(angle, unit="deg")
     aladin.fov = angle_fov
     assert aladin.fov.deg == angle_fov.deg
+
+
+test_stcs_iterables = [
+    "CIRCLE ICRS 258.93205686 43.13632863 0.625",
+    [
+        "POLYGON 257 38 261.005016 50.011125 278.305761 46.00127 257 38",
```

Comment:
Could you add a frame to make it a valid STC-S? Aladin-Lite is nice and accept it for now with the Javascript parser, but we might switch to the Rust parser one day (more complete, but also more strict).